src/js/utils/captions.js

```diff
@@ -22,9 +22,15 @@ export const splitTextTracksStr = (textTracksStr = '') =>
  * @returns {Object} An object that resembles a (partial) TextTrack (`{ language: string; label?: string; }`)
  */
 export const parseTextTrackStr = (textTrackStr = '') => {
-  const [language, encodedLabel] = textTrackStr.split(':');
+  let [kind, language, encodedLabel] = textTrackStr.split(':');
```

Comment:
question(blocking): given our new use case/merged types, can't we currently assume we'll always have a `kind`? If yes, this feels like it could be significantly simplified.